CHANGELOG_iOS.md

```diff
@@ -1,5 +1,28 @@
 # Changelog
 
+## [1.66.113](https://github.com/brave/brave-browser/releases/tag/v1.66.113)
+
+### Web3
+
+ - Added support to retry, cancel and speed up transactions. ([#36627](https://github.com/brave/brave-browser/issues/36627))
+ - Fixed toggle being missing when importing an existing Wallet. ([#38123](https://github.com/brave/brave-browser/issues/38123))
```

Comment:
`Fixed missing import legacy wallet option when when restoring wallet in onboarding`